modules/iam-user/outputs.tf

```
@@ -149,3 +149,8 @@ output "iam_user_ssh_key_fingerprint" {
   description = "The MD5 message digest of the SSH public key"
   value       = try(aws_iam_user_ssh_key.this[0].fingerprint, "")
 }
+
+output "custom_iam_policy_arns" {
```

```suggestion
output "policy_arns" {
```

The word `custom` is obsolete in this context. Please change the input variable name as well.